boards/arm/qemu_cortex_m4/doc/index.rst

```diff
@@ -0,0 +1,102 @@
+.. _qemu_cortex_m4:
+
+ARM Cortex-M4 Emulation (QEMU)
+##############################
+
+Overview
+********
+
+This board configuration will use QEMU to emulate the STM32F405 platform.
+
+.. figure:: qemu_cortex_m4.png
+   :width: 600px
+   :align: center
+   :alt: Qemu
+
+   Qemu (Credit: qemu.org)
+
+This configuration provides support for an ARM Cortex-M4 CPU and these devices:
+
+* Nested Vectored Interrupt Controller
+* System Tick System Clock
+* Stellaris UART
```

Comment:
Should we add the floating-point co-processor?

Reply:
Done